src/dmd/cli.d

```diff
@@ -471,7 +471,7 @@ dmd -cov -unittest myprog.d
             $(DT native)$(DD use the architecture the compiler is running on)
             )`,
         ),
-        Option("mcpu=?",
+        Option("mcpu=[h|help]",
```

Comment:
Because I wanted to phase out `?` as it's problematic on many shells. Anyhow I now changed it to: `mcpu=h|help|?`, s.t. it's still mentioned but not as first choice.